### PR TITLE
Minor text changes [ci skip]

### LINKED ISSUE #391: Misleading step name when multiple master nodes are being smoke-tested
After the initial installation of a multi-master cluster, we run the smoke test on each node serially.

Currently, the output looks like this:
```
Running Smoke Test==================================================================
Smoke Test Master Nodes                                                         [OK]
Smoke Test Master Nodes                                                         [OK]
```

This is misleading as it implies the masters are being smoke-tested multiple times.

### PR DESCRIPTION
**Pull request by dkoshkin**
Fixes  #391
Some minor text changes.

### CODE REVIEW

File: ansible/_smoketest.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
   - hosts: master
     any_errors_fatal: true
-    name: "Smoke Test Master Nodes"
+    name: Smoke Test Master Node
```
Comment (alexbrand) [inline ansible/_smoketest.yaml]: Can we print the host name here?

Comment (dkoshkin) [inline ansible/_smoketest.yaml]: No unfortunately, the variable does not get substituted.

Comment (alexbrand) [inline ansible/_smoketest.yaml]: Bummer. Oh well, it is what it is.

**Comment by alexbrand** [comment]
LGTM

**Pull request merged**